src/components/UI/Cards/VideoCards/VideoCards.css

```diff
@@ -84,6 +84,21 @@ a{
 	width: 100%;
 }
 
+.video-badge-container{
+	height: fit-content !important;
```

Comment:
suggestion: align codes and check whether the usage of !important is necessary or not

Reply:
I actually did that but the hierarchy of structure is not getting me result for this so need to do inportant